wagtail/admin/templates/wagtailadmin/pages/listing/_locked_indicator.html

```diff
@@ -7,6 +7,11 @@
 {% endcomment %}
 
 {% if page.locked %}
-    {# TODO: Different look when it's locked by current user? #}
-    <span class="indicator locked-indicator icon icon-locked" title="{% trans 'This page is locked to further editing' %}"></span>
+    <span class="indicator locked-indicator {% if page.owner == request.user %}locked-indicator-owner{% endif %} icon icon-locked"
```

Comment:
Instead of the `title` attribute - let's use an inner span.ghat has the class `w-sr-only` (this should E better supported for screen readers).

Reply:
An alternative approach would be to use a tooltip - this way we keep the ability to show the label for non screen readers without using title which is not supported on some mobiles. 

See the 'last updated' html for how this is done.